src/gui/dialogs/unit_attack.cpp

```diff
@@ -38,13 +38,15 @@ REGISTER_DIALOG(unit_attack)
 unit_attack::unit_attack(const unit_map::iterator& attacker_itor,
 						   const unit_map::iterator& defender_itor,
 						   std::vector<battle_context>&& weapons,
-						   const int best_weapon)
+						   const int best_weapon,
+						   std::vector<gui2::widget_data>& data_vector)
```

Comment:
This is a really generic name. It should be changed to a name that more specifically describes what this variable holds.